ocf_datapipes/select/select_t0_time.py

```diff
@@ -41,15 +44,25 @@ def __iter__(self) -> pd.Timestamp:
 
             if self.return_all_times:
 
+                number_of_locations = next(self.number_locations_datapipe)
```

Comment:
If this is None, what happens?

Reply:
yea, it wont work, ill probably put a more distinct error message

Reply:
Would you be able to change it so if it is None, then it just does return one location each time? Its how I use it in the MetNet validation for example

Reply:
yep, ill do that